sdk/core/Azure.Core.TestFramework/src/TestLogger.cs

```diff
@@ -56,15 +51,16 @@ internal class TestLogger : IDisposable
             _log = new StringBuilder();
 
         /// <summary>
-        /// Output the Events to the console in the case of test failure.
+        /// Output the Events to the NUnit TestContext. For failed tests,
+        /// the output will be available in the individual tests attachment in
+        /// Azure Dev Ops. For passed tests, you will need to download the parent test run
+        /// attachments which will give you the trx files for all tests. These files
+        /// can be opened in VS to review the logs.
         /// This will include the HTTP requests and responses.
         /// </summary>
         public void OutputEventsForTest()
         {
-            if (_verbose || TestContext.CurrentContext.Result.Outcome.Status == TestStatus.Failed)
-            {
-                TestContext.Out.WriteLine(_log.ToString());
-            }
+            TestContext.Out.WriteLine(_log.ToString());
```

Comment:
Is this any better than just logging to stdout? I know we talked about problems with that, but I thought `TestContext.Out` had the same problems. If not better, it's pointless to maintain a separate logger (than the console logger) that uses increased memory and time to reformat the payload and store it in an expanding `StringBuilder`, just to attach it as stdout anyway.

Reply:
Logging to STDOUT is going through another layer of indirection just to get to `TestContext.Out`. 

We should keep the logger type because we are going to add more stuff to it (like outputting to a file)